tools/docker/Dockerfile

```diff
@@ -264,11 +264,22 @@ SHELL ["/bin/bash", "-c"]
 # Test image that starts up containerd and the naive snapshotter. The default CMD will drop to a bash shell. Overrides
 # to CMD will be provided appended to /bin/bash -c
 FROM firecracker-containerd-unittest as firecracker-containerd-e2etest-naive
+# Install tini init to handle zombies created by our double-forked shims.
+# Unfortunately, we need to do this rather than only use docker's "--init" flag because docker
+# currently puts its default init binary at /dev/init, but we need to bind-mount /dev from
```

Comment:
Would it make sense to mount our `/dev` to a different path and then copy the contents to `/dev`?

Reply:
I don't think that will work here as we want the actual bind mount of /dev from the host so if udev creates a device, our container will also see it. If we just copied when the container starts we will just be getting a snapshot of what /dev looked like at that time rather than ongoing updates to it.